profile/README.md

```diff
@@ -93,6 +93,36 @@ jobs:
           command: myscript
 ```
 
+### Specify MATLAB Version on Self-Hosted Runner
+When you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** action in your workflow, the self-hosted runner uses the topmost MATLAB version on the system path. The build fails if the runner cannot find any version of MATLAB on the path.
+
+You can prepend your preferred version of MATLAB to the PATH environment variable of the runner. For example, prepend MATLAB R2023a to the path and use it to run your script.
+
+```YAML
+name: Run MATLAB Script on Self-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: self-hosted
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3 
+      - name: Prepend MATLAB to PATH on Windows (PowerShell)
+        if: runner.os == 'Windows'
+        run: echo "C:\Program Files\MATLAB\R2023a\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append     
+      - name: Prepend MATLAB to PATH on Linux
+        if: runner.os == 'Linux'
+        run: echo "/usr/local/MATLAB/R2023a/bin" >> $GITHUB_PATH
```

Comment:
@mcafaro: I have access to a Windows runner only. Therefore, I haven't been able to verify that the code on line 116 or 118 executes as expected.